sdk/cosmos/azure_data_cosmos/README.md

```diff
@@ -1,5 +1,200 @@
+
+async fn example(cosmos_client: azure_data_cosmos::CosmosClient) -> Result<(), Box<dyn std::error::Error>> {
+    let properties = ContainerProperties {
+        id: "aContainer".into(),
+        partition_key: PartitionKeyDefinition::new(vec!["/id".into()]),
+        ..Default::default()
+    };
+    let options = CreateContainerOptions {
+        throughput: Some(ThroughputProperties::manual(400)),
+        ..Default::default()
+    };
+    let database = cosmos_client.database_client("myDatabase");
+    let response = database.create_container(properties, Some(options)).await?;
+    Ok(())
+}
+```
+
+### CRUD operation on Items
+
+```rust
+use serde::{Serialize, Deserialize};
+use azure_data_cosmos::models::PatchDocument;
+
+#[derive(Serialize, Deserialize)]
+struct Item {
+    pub id: String,
+    pub partition_key: String,
+    pub value: String,
+}
+
+async fn example(cosmos_client: azure_data_cosmos::CosmosClient) -> Result<(), Box<dyn std::error::Error>> {
+    let item = Item {
+        id: "1".into(),
+        partition_key: "partition1".into(),
+        value: "2".into(),
+    };
+
+    let container = cosmos_client.database_client("myDatabase").container_client("myContainer");
+
+    // Create an item
+    container.create_item("partition1", item, None).await?;
+
+    // Read an item
+    let item_response = container.read_item("partition1", "1", None).await?;
+    let mut item: Item = item_response.into_json_body().await?;
+
+    item.value = "3".into();
+
+    // Replace an item
+    container.replace_item("partition1", "1", item, None).await?;
+
+    // Patch an item
+    let patch = PatchDocument::default()
+        .with_add("/newField", "newValue")?
+        .with_remove("/oldFieldToRemove")?;
+
+    container.patch_item("partition1", "1", patch, None).await?;
+
+    // Delete an item
+    container.delete_item("partition1", "1", None).await?;
+    Ok(())
+}
+```
+
+## Next steps
+
+- [Resource Model of Azure Cosmos DB Service](https://learn.microsoft.com/azure/cosmos-db/sql-api-resources)
+- [Azure Cosmos DB Resource URI](https://learn.microsoft.com/rest/api/documentdb/documentdb-resource-uri-syntax-for-rest)
+- [Partitioning](https://learn.microsoft.com/azure/cosmos-db/partition-data)
+- [Using emulator](https://github.com/Azure/azure-documentdb-dotnet/blob/master/docs/documentdb-nosql-local-emulator.md)
+
+## Next steps
+
+### Client library support
+
+Client and management libraries <!-- TODO: Update link and uncomment when Rust SDK has a page on the releases site.> listed on the [Azure SDK release page](https://azure.github.io/azure-sdk/releases/latest/python.html)</!--> that support Microsoft Entra authentication accept credentials from this library. You can learn more about using these libraries in their documentation, which is <!-- TODO: uncomment when Rust SDK has a release page.>linked from the release page</!-->available at [Docs.rs](https://Docs.rs/azure_data_cosmos).
```

Comment:
This seems like it should be on the identity README, not ours?

Reply:
Yes, this is specific to the Identity library. Please remove it.

Reply:
We have a standard blurb for this that links to Identity. See KV secrets's README and just copy that as needed.